core/src/main/java/org/elasticsearch/search/suggest/phrase/DirectCandidateGeneratorBuilder.java

```diff
@@ -474,9 +474,7 @@ static StringDistance resolveDistance(String distanceVal) {
             return new LevensteinDistance();
         } else if ("levenshtein".equals(distanceVal)) {
             return new LevensteinDistance();
-            // TODO Jaro and Winkler are 2 people - so apply same naming logic
-            // as damerau_levenshtein
-        } else if ("jarowinkler".equals(distanceVal)) {
+        } else if ("jaro_winkler".equals(distanceVal) || "jarowinkler".equals(distanceVal)) {
```

Comment:
Do we really need to support both? Can we settle on one (with the underscore), and deprecate the other? And same for `damerau_levenshtein`?

Reply:
+1 for deprecation of `jarowinkler`.

Not sure about that score creep tho :)

Reply:
Agree, the other should be separate.